pinery-client/src/main/java/ca/on/oicr/pinery/client/PineryClient.java

```diff
@@ -69,8 +68,7 @@ public PineryClient(String baseUrl, boolean ignoreHttpsWarnings) {
         ignoreHttpsWarnings ? PineryClient.getInsecureClient() : PineryClient.getSecureClient());
   }
 
-  @VisibleForTesting
-  protected PineryClient(String baseUrl, Client client) {
+  public PineryClient(String baseUrl, Client client) {
```

Comment:
Should remain protected and commented to indicate it's only intended for testing

Reply:
Should it though? I'm unclear why this method was ever protected. It's not like it is dangerous to use. Since is protected and the class isn't final, it's effectively public.

Reply:
Not dangerous, just unintended. I suppose it doesn't matter